@mountiny Can you re-run the podfile test?
npm ERR! 504 Gateway Timeout - GET https://gitpkg.vercel.app/facebook/flipper/react-native/react-native-flipper?9cacc9b59402550eae866e0e81e5f0c2f8203e6b

--- a/src/components/withWindowDimensions/index.native.js
+++ b/src/components/withWindowDimensions/index.native.js
@@ -65,7 +65,7 @@ function WindowDimensionsProvider(props) {
         <SafeAreaInsetsContext.Consumer>
             {(insets) => {
                 const isExtraSmallScreenWidth = windowDimension.windowWidth <= variables.extraSmallMobileResponsiveWidthBreakpoint;
-                const isSmallScreenWidth = !isExtraSmallScreenWidth;
+                const isSmallScreenWidth = true;
                 const isMediumScreenWidth = false;
                 const isLargeScreenWidth = false;
                 return (

--- a/src/hooks/useWindowDimensions/index.native.js
+++ b/src/hooks/useWindowDimensions/index.native.js
@@ -9,7 +9,7 @@ import variables from '../../styles/variables';
 export default function () {
     const {width: windowWidth, height: windowHeight} = useWindowDimensions();
     const isExtraSmallScreenHeight = windowHeight <= variables.extraSmallMobileResponsiveHeightBreakpoint;
-    const isSmallScreenWidth = !isExtraSmallScreenHeight;
+    const isSmallScreenWidth = true;
     const isMediumScreenWidth = false;
     const isLargeScreenWidth = false;
     return {

